lib/graphql_devise/concerns/controller_methods.rb

```diff
@@ -0,0 +1,64 @@
+module GraphqlDevise
+  module Concerns
+    module ControllerMethods
+      extend ActiveSupport::Concern
+
+      private
+
+      def raise_user_error(message)
+        raise GraphqlDevise::UserError, message
+      end
+
+      def raise_user_error_list(message, errors:)
+        raise GraphqlDevise::DetailedUserError.new(message, errors: errors)
+      end
+
+      def remove_resource
+        controller.resource = nil
+        controller.client_id = nil
+        controller.token = nil
+      end
+
+      def request
+        controller.request
+      end
+
+      def response
+        controller.response
+      end
+
+      def controller
+        context[:controller]
+      end
+
+      def resource_class
+        context[:resource_class]
+      end
+
+      def recoverable_enabled?
+        resource_class.devise_modules.include?(:recoverable)
+      end
+
+      def confirmable_enabled?
+        resource_class.devise_modules.include?(:confirmable)
+      end
+
+      def current_resource
+        context[:current_resource]
+      end
+
+      def client
```

Comment:
Since we have the `client` method here, what about moving the `client_and_token` method as well?